package.json

```diff
@@ -41,6 +41,7 @@
     "eslint": "7.27.0",
     "jest": "27.0.1",
     "memorystream": "^0.3.1",
+    "normalize-url": "6.0.1",
```

Comment:
Is this needed as a `devDep`?

Reply:
Nope, that one was added by accident. Thx a lot for noticing!